_data/presentations.yml

```diff
@@ -4,4 +4,11 @@
   title: "Agent-based Approaches for Off-Normal Modeling of Fuel Cycles: Disruptions and Diversions"
   url: https://docs.google.com/presentation/d/e/2PACX-1vR0VuXn9tlpcxJFQUMcSb7atU45Qe85LO1dAViA-gCe36JOMXk4AShU1m8RCa3FLEiSs-UvaDmBMfWv/pub
   person: Paul Wilson
-  person_dir: pphw
\ No newline at end of file
+  person_dir: pphw
+
+- date: "2022-10-04"
+  venue: Gateway for Accelerated Innovation in Nuclear (GAIN)
+  venue_url: https://gain.inl.gov/SitePages/Home.aspx
+  title: "Investigating Benefits and Challenges of Converting Coal Power Plants to Nuclear Power Plants"
+  person: Katie Biegel
```

Comment:
How about adding this as a `url:` https://gain.inl.gov/SitePages/Coal2Nuclear.aspx?

Reply:
Makes sense! Added